### ISSUE TITLE
Contain of "About" is not reloaded when context change

### ISSUE THREAD

**Issue by mikesmichael** [infra-geo-ouverte/igo2-lib#266]
**Please tell us about your environment:**
Windows 8, Chrome version 72.0.3626.96 

* **Igo Version:**
0.23.1

I put a different html value for the about in a new context. The html is not loaded when I change to this context. The contain stay to the value defined in the default context.

**Comment by pelord** [comment]
I've found the problem. 
[../projects/context/src/lib/context-manager/shared/tool-context.directive.ts](https://github.com/infra-geo-ouverte/igo2-lib/blob/6bb9a66b3f00587488ae5dcfb33bf39b522cd413/projects/context/src/lib/context-manager/shared/tool-context.directive.ts#L45)
The app reuse the previously provided tool (options, name, icon...) if the name of the tool match...

I found a simple fix but do not cover all possibilities. 

I think we need to create a tool-context.state to keep values provided by context.json

Here the plan:

1. Solve this issue with a simple fix. In this case, if the new handled context do not have options, the will reuse those provided previously.
```
if (tool_.options) {
tool.options = tool_.options;
}
````
2. Create a state component to hold tool options and properties...

I don't have much time to work on it a this time. 

Waiting for your comments.
